R/cqcp_helper.R

```diff
@@ -56,11 +56,12 @@ cqcp_colourise <- function(str, colour) {
 cqcp_check_input <- function(data, print = TRUE, file = NULL, as_list = FALSE){
   
   ok <- TRUE
-  ch_1a <- ch_1b <- ch_2 <- ch_3 <- ch_4 <- ch_5 <- TRUE
+  ch_1a <- ch_1b <- ch_1c <- ch_2 <- ch_3 <- ch_4 <- ch_5 <- TRUE
+  #mess_1a <- mess_1b <- mess_1c <- mess_2 <- mess_3 <- mess_4 <- mess_5 <- cqcp_colourise(paste0("     ! Check not carried out.\n"), "red")
```

Comment:
Is this supposed to be commented out? Might as well removed it then...

Reply:
Thanks! Was included during development. Removed now.